pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
@obastemur Thanks for your updates!

> In this case, isn't _Is_integral is defined twice for signed and unsigned long when BIT64 is set? See line 311 pal/inc/type_traits

As I've mentioned, there was existing fix for that, see line 309. Your update conflicts with cmake build on Linux as we have both ```BIT64``` and ```PLATFORM_UNIX``` defined. Both line 309 and your update get rid of one specialization. Admittedly I think your change is better than existing line 309.

To make this work, please either define ```PLATFORM_UNIX``` in your build and not changing this file, or replace (remove) existing fix on line 309 with your update.